### PR TITLE
Adds a client to publish static set of events to Azure resources

### PR DESCRIPTION
**Pull request by tishan89**
This PR Adds a client to publish static set of events to Azure resources. In this milestone client is configurable via environment variables and following environment variables are available. Please refer ReadMe for more information

### CODE REVIEW

File: component/publisher-client/src/main/java/org/wso2/am/analytics/publisher/client/EventHubClient.java [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        df.setTimeZone(tz);
+        for (int i = 0; i < eventCount; i++) {
+            JsonObject jsonObject = new JsonObject();
+            jsonObject.addProperty("requestTimestamp", df.format(new Date()));
+            jsonObject.addProperty("correlationId", String.valueOf(UUID.randomUUID()));
+            jsonObject.addProperty("keyType", DataHolder.KEY_TYPE[i % 5]);
+            jsonObject.addProperty("nodeId", DataHolder.NODE_ID[i % 5]);
+            jsonObject.addProperty("deploymentId", DataHolder.DEPLOYMENT_ID[i % 5]);
+            jsonObject.addProperty("apiId", DataHolder.API_UUID[i % 5]);
+            jsonObject.addProperty("subscriber", DataHolder.SUBSCRIBER[i % 5]);
```
Comment (fazlan-nazeem) [inline component/publisher-client/src/main/java/org/wso2/am/analytics/publisher/client/EventHubClient.java:89]: subscriber not needed right?

**Comment by fazlan-nazeem** [comment]
Shall we add README on how to execute this?

File: component/publisher-client/src/main/java/org/wso2/am/analytics/publisher/client/EventHubClient.java (hunk 1 above)
Comment (tishan89) [reply component/publisher-client/src/main/java/org/wso2/am/analytics/publisher/client/EventHubClient.java]: removed

**Comment by tishan89** [comment]
> Shall we add README on how to execute this?

Already added

**Review by fazlan-nazeem — APPROVED** [review]

**Pull request merged**